app/src/main/java/com/vmware/nimbus/api/APIService.java

```diff
@@ -158,4 +162,10 @@ public Color getPowerState(final DeploymentCallback callback, int index) {
         }
         return null;
     }
+
+    // Displays a toast so we can verify that the buttons work when clicked
+    public static void toastMsg(String msg, Context c) {
+        Toast toast = Toast.makeText(c, msg, Toast.LENGTH_SHORT);
```

Comment:
Error messages can be `LENGTH_LONG` because we want the user to be able to visually process the error message.